socorro/unittest/middleware/test_middleware_app.py

```diff
@@ -589,8 +596,17 @@ def put(self, server, url, data=None, expect_errors=False):
 
     def test_overriding_implementation_class(self):
         config_manager = self._setup_config_manager({
-            'implementations.service_overrides': 'Backfill: psql, Bugs: typo'
-        })
+            'implementations.implementation_list': (
+                'psql:socorro.unittest.middleware'
+            ),
+            'implementations.service_overrides': (
+                'Bugs: typo'
+            )
+        },
+            services_list=(
+                ('/bugs/', 'bugs.Bugs'),
+            )
```

Comment:
The problem to me is that you are changing the original code (making ``SERVICES_LIST`` a class attribute) just for the sake of our tests, and I don't like that.